S23/kishore223/lowercase.py

```diff
@@ -0,0 +1,80 @@
+import unittest
+
+def lowercase_word(word, lang):
+    if lang in ['tr', 'az']:
```

Comment:
If "lang" is meant to be a BCP-47 code, you'll need to pick out the language code from the beginning.